federatedscope/mf/trainer/trainer_sgdmf.py

```diff
@@ -37,13 +37,13 @@ def init_sgdmf_ctx(base_trainer):
     ctx = base_trainer.ctx
     cfg = base_trainer.cfg
 
-    sample_ratio = float(cfg.data.batch_size) / cfg.model.num_user
+    sample_ratio = float(cfg.dataloader.batch_size) / cfg.model.num_user
     # Noise multiplier
     tmp = cfg.sgdmf.constant * np.power(sample_ratio, 2) * (
         cfg.federate.total_round_num * ctx.num_total_train_batch) * np.log(
             1. / cfg.sgdmf.delta)
     noise_multipler = np.sqrt(tmp / np.power(cfg.sgdmf.epsilon, 2))
-    ctx.scale = max(cfg.sgdmf.theta, 1.) * noise_multipler * np.power(
+    ctx.scale = max(cfg.dataloader.theta, 1.) * noise_multipler * np.power(
```

Comment:
Since `theta` is only used in sgdmf, is it appropriate to place `theta` under the namespace of dataloader? (that all users can see it)

Reply:
We can add a docstring to explain this, as many other args (`sizes` is graph-related) in `dataloader` are optional.